### PR TITLE
Authorized navigation

### PR DESCRIPTION
**Pull request by emrerdem1**
Hey Louis!

I was waiting for demo to PR this changes. They actually does not have the new features, just popover notifications based on login states.

If user logged in, they will use a different user navigation form.

Please let me know if you see anything wrong.

### CODE REVIEW

File: src/components/layout/nav/Navbar.jsx [hunk 1]
```diff
@@ -138,13 +164,52 @@ const Navbar = () => {
...
+    <NavLink to="/shoppingcart">
+      <i className="fas fa-shopping-cart"></i>
+    </NavLink>
+  );
+
+  const handleShoppingCartWarning = (e) => {
+    setShoppingCartTargetUi(e.target);
+    setIsShoppingCartAccesRejected(true);
+    handleStatus(CLICK_AWAY);
+    setTimeout(() => setIsShoppingCartAccesRejected(false), ALERT_OPEN_SECONDS);
```
Comment (louisrli) [inline src/components/layout/nav/Navbar.jsx]: typo in access

File: src/components/layout/nav/NavigationTabOnLogin.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
```
Comment (louisrli) [inline src/components/layout/nav/NavigationTabOnLogin.jsx:1]: pls write a comment at the top of this file describing what it does

File: src/components/layout/nav/NavigationTabOnLogin.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import { useSelector, useDispatch } from "react-redux";
+import { logOut, popUpStatus } from "../../redux/actions/index";
+
+const NavigationTabOnLogin = () => {
+  const history = useHistory();
+  const isAdmin = useSelector((state) => state.authentication.isAdmin);
+  const dispatch = useDispatch();
+  const loggingOut = () => {
+    dispatch(logOut());
+    isAdmin && history.push("/");
```
Comment (louisrli) [inline src/components/layout/nav/NavigationTabOnLogin.jsx]: do not do this

Comment (louisrli) [inline src/components/layout/nav/NavigationTabOnLogin.jsx]: do not do this

**Review by louisrli — APPROVED** [review]

File: src/components/layout/nav/NavigationTabOnLogin.jsx (hunk 3 above)
Comment (emrerdem1) [reply src/components/layout/nav/NavigationTabOnLogin.jsx]: Okay, I'm changing it to normal `if` statement.

**Review by MiraAr — APPROVED** [review]

File: src/components/layout/nav/NavigationTabOnLogin.jsx (hunk 3 above)
Comment (louisrli) [reply src/components/layout/nav/NavigationTabOnLogin.jsx]: please don't do this again, if you need further reasoning you may consult the below

https://stackoverflow.com/questions/34704750/one-line-if-vs-in-javascript

Comment (emrerdem1) [reply src/components/layout/nav/NavigationTabOnLogin.jsx]: `Remember that _you six months from now_ is _someone else_.` 💯 

I see, then I'm gonna use it only in `return` statement of component from now onwards.

**Pull request merged**